### PR TITLE
Projects structure and Some components

### PR DESCRIPTION
**Pull request by isneezy**
Solves #1 
Forgive me if this PR does not conform to Guide Lines, it contains many unrelated commits.
I created the project structure, added a sub-project to serve as example site, and created some components of which some still need a lot of improvement.
I was just rushing to get a start page identical example of the Tabler UI page

### CODE REVIEW

**Comment by MUCHZER** [comment]
Lots to improve but look like a good start! Thank you

**Comment by isneezy** [comment]
As I stated at https://github.com/tabler/tabler-vue/issues/1#issuecomment-393564893, I am in a rush with one of my projects and it depends on this project. I know the correct thing would be for me to stop and wait for review, approval etc... but I can not. I've been working on some more components, I would like to know if I open another pull request or can I add everything here?

**Comment by Nilpo** [comment]
@isneezy Add everything here for now. There are also quite a few remnants from the tabler-react project that can be removed.

**Comment by isneezy** [comment]
> There are also quite a few remnants from the tabler-react project that can be removed.
Could you share so i can try to follow from now on?

**Comment by Nilpo** [comment]
This will give us something to start with. Good work.

File: README.md [hunk 1]
```diff
@@ -13,9 +13,23 @@ TODO: Describe the installation process
...
 Contributions are welcome. See [CONTRIBUTING.md](CONTRIBUTING.md).
 
+The package is made up of 2 main folders:
+- /src
+- /example is a [vue-cli webpack](https://github.com/vuejs/vue-cli) based demo website
+
+To setup and run a local copy:
+1. Clone this repo with `git clone https://github.com/tabler/tabler-vue.git`
+2. Run `yarn install` in root folder
+3. Run `Yarn dev`
```
[the remark this answers is not in the document]
Comment (isneezy) [reply README.md]: Ups! My bad, I didn't notice that! This may lead to false positive issues for "copy paste" apologists! We should fix ASAP

**Pull request merged**